### DISCUSSION TITLE
black screen when starting (after installing)

### DISCUSSION THREAD

**Discussion by timurchikk** [utmapp/UTM#4811] [UTM Help (macOS)] [answered]
**Configuration**  
* UTM Version: 4.0.9
* macOS Version: 13.0
* Mac Chip: M2

* Issue:  i clear cash and remove serial from machine, and after i starting kali, showing start choices like, kali gnu/linux, Advanced options etc, and 5, 4, 3 seconds also, but when i starting the machine i getting only black screen

<img width="1470" alt="Screenshot 2022-12-14 at 6 08 42 PM" src="https://user-images.githubusercontent.com/85063394/207617613-3dc777ac-96c3-41b6-be64-999592e1cb28.png">

**Comment by osy** [comment, marked as answer]
https://docs.getutm.app/guides/kali/#black-screen-on-start
